modules/peridynamics/src/mesh/PeridynamicsMesh.C

```diff
@@ -517,6 +520,22 @@ PeridynamicsMesh::getBoundaryOffset(dof_id_type node_id)
     return 0.0;
 }
 
+Real
+PeridynamicsMesh::getNeighborWeight(dof_id_type node_id, dof_id_type neighbor_id)
+{
+  if (node_id > _n_pdnodes)
```

Comment:
Are these the kinds of errors that would not be caused by user error, but would rather be due to a coding error? If it's the latter, I think using mooseAssert() would be appropriate because it would avoid the run-time cost of doing these checks (not that they're particularly expensive).

Reply:
I just updated all the similar checks in the same class.